lionblog-eunjae/src/routes/PostCreatePage.jsx

```diff
@@ -1,7 +1,26 @@
 const PostCreatePage = () => {
+	const handlePostCreatePageSubmit = () => {
```

Comment:
나중에는 실제로 입력하는 내용을 반영하여 post 객체를 submit해야하기 때문에 차후에는 상단에 post 정의를 해주는 것도 좋을 것 같아요!
```suggestion
	const post = {
    id: posts.length,
    title: "",
    content: "",
    author: { id: posts.length, username: "아기사자" },
    tags: [],
    like_users: [],
    created_at: "2024-02-04T07:42:50.658501Z",
  };
```